If_time/Program.cs

```diff
@@ -6,38 +6,69 @@ class Program
     {
         static void Main(string[] args)
         {
-           TimeSpan current_time = DateTime.Now.TimeOfDay;
-
-Console.WriteLine( current_time);
+          Console.WriteLine("Enter your name:");
+          string Name= Console.ReadLine();     
+          Console.Clear();
+          //var btn= ConsoleKey.Escape;
+          ConsoleKeyInfo cki= new ConsoleKeyInfo();
+          
+       while (true)
+       {
+           
+      Console.Clear();
+                         
+          TimeSpan current_time = DateTime.Now.TimeOfDay;
+          Console.WriteLine($"current time {current_time.ToString(@"hh\:mm\:ss")}");
+        
 
 int current_hour = current_time.Hours;
 
 if (current_hour>=9 & current_hour<12)

```

Comment:
As you remember, I mentioned that we need to use logic AND operator such as `&&`.
And I will ask you WHY.